tests/test_configuration_parser.py

```diff
@@ -92,6 +96,66 @@ def test_normalize_config(self):
             newdata = self.parser.normalize_config()
             ok_('use_scaled_predictions' not in newdata.keys())
 
+    def test_load_config_from_dict(self):
+        data = {'expID': 'test'}
+        configdir = abspath('/path/to/configdir')
```

Comment:
Isn't this already an absolute path?

Reply:
Windows resolves them with `D:` so without this tests fail on Azure.